nexus/examples/qmcpack/rsqmc_misc/estimators/iron_ldaU_dmc.py

```diff
@@ -0,0 +1,220 @@
+from nexus import generate_qmcpack,vmc,dmc
+from nexus import ppset,obj
+
+settings(
+    pseudo_dir    = './pseudopotentials',
+    results       = '',
+    sleep         = 3,
+    machine       = 'ws16',
+    status_only   = 0,
+    generate_only = 1,
+    )
+
+ppset(
+    label   = 'ccECP-soft',
+    pwscf   = 'Fe.ccECP-soft.upf'.split(),
+    qmcpack = 'Fe.ccECP-soft.xml'.split(),
+    )
+pseudos = 'ccECP-soft'
+
+# inputs shared by all dft calculations
+dft_shared = obj(
+    input_type       = 'generic',
+    input_dft        = 'lda',
+    ecutwfc          = 400,
+    bandfac          = 1.5,    # Adjust as needed for the 1RDM orbital basis
+    occupations      = 'smearing',
+    smearing         = 'fermi-dirac',
+    degauss          = 1e-3,
+    mixing_beta      = 0.4,
+    nspin            = 2,
+    U_projection_type = 'ortho-atomic',
+    hubbard_u        = obj(Fe=2.0),
+    start_mag        = obj(Fe=0.1),
+    kshift           = (0,0,0),
+    pseudos          = pseudos,
+    )
+
+# inputs shared by all qmc calculations
+qmc_shared = obj(
+    driver         = 'batched',
+    input_type     = 'basic',
+    meshfactor     = 1.00,
+    spin_polarized = True,
+    pseudos        = pseudos,
+    )
+
+# read the primitive cell
+structure = read_structure('Fe_bcc.xsf')
+structure.change_units('B')
+
+system = generate_physical_system(
+    structure  = structure.copy(),
+    kgrid      = (1,1,1),
+    kshift     = (0,0,0),
+    net_spin   = 6,
+    net_charge = 0,
+    symm_kgrid = False,
+    tiling     = (1,1,1),
+    Fe         = 16,
+    )
+
+basepath = './'
+
+scf = generate_pwscf(
+    identifier        = 'scf',
+    path              = basepath + 'scf',
+    job               = job(cores=16, app='pw.x'),
+    calculation       = 'scf',
+    system            = system,
+    kgrid             = (24,24,24),
+    **dft_shared
+    )
+
+nscf = generate_pwscf(
+    identifier        = 'nscf',
+    path              = basepath + 'nscf',
+    job               = job(cores=16, app='pw.x'),
+    calculation       = 'nscf',
+    system            = system,
+    nosym             = True,
+    dependencies      = (scf,'charge_density'),
+    **dft_shared
+    )
+
+p2q = generate_pw2qmcpack(
+    identifier        = 'p2q',
+    path              = basepath + 'nscf',
+    job               = job(cores=16,app='pw2qmcpack.x'),
+    write_psir        = False,
+    dependencies      = (nscf,'orbitals'),
+    )
+
+optJ12 = generate_qmcpack(
+    identifier           = 'optJ12',
+    path                 = basepath + 'optJ12',
+    job                  = job(cores=16,threads=4,app='qmcpack'),
+    system               = system,
+    twistnum             = 0,
+    qmc                  = 'opt',
+    J2                   = True,
+    minmethod            = 'oneshift',
+    warmupsteps          = 10,
+    init_cycles          = 5,
+    cycles               = 5,
+    walkers_per_rank     = 256,
+    samples              = 1e5,
+    blocks               = 10,
```

Comment:
Favor more blocks here so people can make a statistical analysis of the optimization afterwards.

Reply:
Done